be/src/storage/rowset/segment_v2/bitshuffle_page.h

```diff
@@ -387,13 +404,14 @@ class BitShufflePageDecoder final : public PageDecoder {
 
     Status _decode_to(uint8_t* to) {
         char* in = const_cast<char*>(&_data[BITSHUFFLE_PAGE_HEADER_SIZE]);
```

Comment:
move this into if (!_options.use_cache) {

Reply:
done